syntax = "proto2";

package jungi.mobgtw.proto;
option optimize_for = LITE_RUNTIME;

message NetworkSettingsResponse {
  optional int32 operation_status = 1;
  optional bool ethernet_state = 2;
  optional int32 wifi_mode = 3;
  optional string wifi_net_name = 4;
  optional string wifi_net_password = 5;
}
